README.md

```diff
@@ -649,7 +649,7 @@ Example of an Alonzo block returned during local chain sync:
 ### Step 1
 Run local ogmios instance:
 ```
-docker-compose up -f deploy/docker-compose.yml -d
+docker-compose -f deploy/docker-compose.yml up -d
```

Comment:
docker-compose seems to have problems parsing parameters, to avoid them up must be after the `-f` option